integration-tests/mail/src/main/java/org/apache/camel/quarkus/component/mail/MockMailbox.java

```diff
@@ -16,40 +16,90 @@
  */
 package org.apache.camel.quarkus.component.mail;
 
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.util.Properties;
+
 import javax.enterprise.context.ApplicationScoped;
+import javax.mail.BodyPart;
+import javax.mail.Message;
+import javax.mail.Multipart;
+import javax.mail.Session;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
+import javax.ws.rs.DELETE;
 import javax.ws.rs.GET;
+import javax.ws.rs.POST;
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
 
 import org.jvnet.mock_javamail.Mailbox;
 
-@Path("/mock/{username}")
+@Path("/mock")
 @ApplicationScoped
 public class MockMailbox {
 
+    @DELETE
+    @Path("/clear")
+    @Produces(MediaType.TEXT_PLAIN)
```

Comment:
It is possible to @Produces with a void method ?

Reply:
Nope - too much copy / paste. I'll remove it.